ci-operator/step-registry/capi-migration-check/capi-migration-check-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: capi-migration-check
+  commands: capi-migration-check-commands.sh
+  from: cli
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  documentation: |-
+    Check MachineAPIMigration functionality works well.
```

Comment:
if we add that we are checking status and paused condition here , will it help others in case something else fails ?
example - `Check MachineAPIMigration functionality works well with synchronized status is true and paused status is false`